web-app/js/portal/PortalEvents.js

```diff
@@ -3,8 +3,6 @@ var PORTAL_EVENTS = {
     SELECTED_LAYER_CHANGED:        "selectedLayerChanged",
     BASE_LAYER_CHANGED:            "baseLayerChanged",
     LAYER_REMOVED:                 "layerRemoved",
-    LAYER_LOADING_START:           "layerLoadingStart",
```

Comment:
While you're at it, also those can be removed:
 * `BEFORE_SELECTED_LAYER_CHANGED`
 * `LOAD_SNAPSHOT`